src/index.html

```diff
@@ -11,7 +11,178 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+        <legend>
+        Personal information
+        </legend>
+        <div>
+          <label for="Surname">
+            Surname:
+          </label>
+          <input id="Surname" type="text"
+          name="Surname"
+          autocomplete="off"
+          required
+          >
+        </div>
+        <div>
```

Comment:
Pay attention to your formatting.
Also don't forget about new line between adjacent elements
```suggestion
        <legend>
        Personal information
        </legend>
        
        <div>
          <label for="Surname">
            Surname:
          </label>
          
          <input 
            id="Surname" 
            type="text"
            name="Surname"
            autocomplete="off"
            required
          >
        </div>
        
        <div>
```